tests/test_http.py

```
@@ -60,46 +61,50 @@
+    assert isinstance(adapter, HTTPAdapter)
+    assert adapter.max_retries.status_forcelist == client.status_forcelist
+
     client.configure(status_forcelist=(200, 201, 202))
     assert client.status_forcelist == (200, 201, 202)
+    adapter = client.sess.adapters["https://"]
+    assert isinstance(adapter, HTTPAdapter)
+    assert adapter.max_retries.status_forcelist == client.status_forcelist
 
 
-def test_backoff_factor(client: Client):
+def test_configure_backoff_factor(client: Client):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_configure_backoff_factor`

Add return type annotation: `None`

(ANN201)